Dockerfile

```diff
@@ -1,31 +1,38 @@
 # BUILD
-FROM golang:1.11-alpine as builder
+FROM golang:1.13-alpine as builder
 
-RUN apk add --no-cache git mercurial 
+RUN apk add --no-cache gcc build-base
 
-ENV p $GOPATH/src/github.com/labbsr0x/bindman-dns-bind9
+RUN mkdir /app
+WORKDIR /app
 
-ADD ./ ${p}
-WORKDIR ${p}
-RUN go get -v ./...
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+RUN go test -v ./...
 
 RUN GIT_COMMIT=$(git rev-parse --short HEAD 2> /dev/null || true) \
  && BUILDTIME=$(TZ=UTC date -u '+%Y-%m-%dT%H:%M:%SZ') \
  && VERSION=$(git describe --abbrev=0 --tags 2> /dev/null || true) \
  && CGO_ENABLED=0 GOOS=linux go build --ldflags "-s -w \
-    -X github.com/labbsr0x/bindman-dns-bind9/src/version.Version=${VERSION:-unknow-version} \
-    -X github.com/labbsr0x/bindman-dns-bind9/src/version.GitCommit=${GIT_COMMIT} \
-    -X github.com/labbsr0x/bindman-dns-bind9/src/version.BuildTime=${BUILDTIME}" \
-    -a -installsuffix cgo -o /bindman-dns-manager src/main.go
+    -X github.com/labbsr0x/bindman-dns-bind9/version.Version=${VERSION:-unknow-version} \
+    -X github.com/labbsr0x/bindman-dns-bind9/version.GitCommit=${GIT_COMMIT} \
+    -X github.com/labbsr0x/bindman-dns-bind9/version.BuildTime=${BUILDTIME}" \
+    -a -installsuffix cgo -o /bindman-dns-bind9 main.go
 
-# PKG
-FROM alpine:latest
+## PKG
+FROM alpine
```

Comment:
It would be good to have a specific image base version to prevent unexpected breaks.